py3status/modules/imap.py

```diff
@@ -34,15 +33,27 @@ class Py3status:
     cache_timeout = 60
     criterion = 'UNSEEN'
     format = 'Mail: {unseen}'
+    # remove this? // use format = '[\?not_zero Mail: {unseen}]'
```

Comment:
this comment can be removed we may as well stick with the situation we have for now